### PR TITLE
Import Berlow sources

### PR DESCRIPTION
**Pull request by madig**
1. Add new grading UFOs and overwrite all existing glyphs directories, groups and kerning with Berlow sources, verbatim.
2. Remove `aalt` feature.
3. Set `.alt` glyphs to not export.

TODO:
- [ ] Unfudge production scripts
    - [ ] Adjust partial instancer/Makefiles to produce graded instances?
    - [ ] Get rid of default/export distinction?
- [x] Reinstate `rvrn` where necessary
- [ ] Recheck glyphOrder and postscriptNames for the final fonts to be imported
- [ ] Check final fonts to be imported for leftover glyphs
- [ ] Update the shaping tests
- [ ] Check fontinfo metadata, especially the interpolatable part
- [ ] ?

### CODE REVIEW

**Comment by chrissimpkins** [comment]
> Adjust partial instancer/Makefiles to produce graded instances?

Unnecessary at the moment.  High priority build format is Roman and Italic variable fonts with all axes.  Not necessary to create new named instances for grades in VF's either.  Let's keep the same named instances that we had as of v3.003.

> Get rid of default/export distinction?

Yes please.  Let's eliminate the `expert` and `default` sub-dir structure. We'll only build one default build target and it isn't necessary to place it in a sub-dir.  We'll keep the static and variable dirs for the separate build formats.  All defined in Makefile variables currently

> Set .alt glyphs to not export

Copy / paste Python list https://github.com/googlefonts/googlesans/issues/211#issuecomment-785516852

> rvrn

Changes in progress and not stable downstream yet

> ?

I'm with you on the `?`  Let's venture into the unknown :)

**Comment by madig** [comment]
K, so I will leave `rvrn` out until I hear otherwise.

**Comment by chrissimpkins** [comment]
Can we agree on the following build artifact paths?  I'll watch for where you end up today and can dig in on GH Actions and build scripting file path updates

### Variable

- `build/GoogleSans/variable/GoogleSans[GRAD,opsz,wght].ttf`
- `build/GoogleSans/variable/GoogleSans-Italic[GRAD,opsz,wght].ttf`

### Static instances

- `build/GoogleSans/static/GoogleSans-Regular.ttf`
- `build/GoogleSans/static/GoogleSans-Italic.ttf`
- `build/GoogleSans/static/GoogleSans-Medium.ttf`
- `build/GoogleSans/static/GoogleSans-MediumItalic.ttf`
- `build/GoogleSans/static/GoogleSans-Bold.ttf`
- `build/GoogleSans/static/GoogleSans-BoldItalic.ttf`
- `build/GoogleSans/static/GoogleSansText-Regular.ttf`
- `build/GoogleSans/static/GoogleSansText-Italic.ttf`
- `build/GoogleSans/static/GoogleSansText-Medium.ttf`
- `build/GoogleSans/static/GoogleSansText-MediumItalic.ttf`
- `build/GoogleSans/static/GoogleSansText-Bold.ttf`
- `build/GoogleSans/static/GoogleSansText-BoldItalic.ttf`

**Comment by madig** [comment]
Can implement tomorrow!

**Comment by chrissimpkins** [comment]
> Can implement tomorrow!

Ty!  Wanted to make sure that you are ok with those paths.  I'll begin to look into the changes outside of the Makefiles

File: qa/check-fea.py [hunk 1]
```diff
@@ -131,7 +128,6 @@
     "numr",
     "ordn",
     "pnum",
-    "rvrn",
```
Comment (chrissimpkins) [inline qa/check-fea.py]: I think that we will have to keep this to support Hebrew rvrn for now (https://github.com/googlefonts/googlesans/issues/214#issuecomment-785435774).  These won't be transitioned to compatible forms in time for the v4.000 release but we'll look into it.  David B is trying to eliminate it altogether in the Latin where we are facing bugs.  I think that more rvrn may be coming in with Armenian?  We may not get away from it altogether.  Let's see.

Comment (madig) [reply qa/check-fea.py]: Ah, right. I guess I'll leave in rvrn for non-LGC.

**Pull request closed without merging**